me.ogarcia.lptk.json

```diff
@@ -0,0 +1,51 @@
+{
+    "id" : "me.ogarcia.lptk",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "lptk",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
```

Comment:
build MUST be offline. This is not allowed.

Reply:
but then you have to generate the manifest for the dependencies:

https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo